pep-0458-tuf-online-keys.rst

```diff
@@ -395,6 +395,208 @@ specifying `delegations`__.
 __ https://github.com/theupdateframework/tuf/tree/develop/tuf#delegations
 
 
+File Formats of the PyPI JSON Metadata
+--------------------------------------
+
+This section presents the format of the JSON metadata files.  Examples of the
+roles and their formats mentioned here are available for review in the
+"pep-0458-repository" subdirectory (alongside the "pep-0458.txt" PEP).
+
+
+root.JSON
+~~~~~~~~~
+
+The root.json file is signed by the root role's keys.  It indicates
+which keys are authorized for all top-level roles, including the root
+role itself.  Revocation and replacement of top-level role keys, including
+for the root role, is done by changing the keys listed for the roles in
+this file.
+
+The format of root.json is as follows:
+
+{
+  "_type" : "Root",
+  "version" : VERSION,
+  "expires" : EXPIRES,
+  "keys" : {
+    KEYID : KEY
+    , ... },
+  "roles" : {
+    ROLE : {
+      "keyids" : [ KEYID, ... ] ,
+      "threshold" : THRESHOLD },
+    ...
+  }
+}
+
+VERSION is an integer that is greater than 0.  Clients MUST NOT replace a
+metadata file with a version number less than the one currently trusted.
+
+EXPIRES determines when metadata should be considered expired and no longer
+trusted by clients.  Clients MUST NOT trust an expired file.
+
+A ROLE is one of "root", "snapshot", "targets", "timestamp", or "mirrors".
+A role for each of "root", "snapshot", "timestamp", and "targets" MUST be
+specified in the key list. The role of "mirror" is optional.  If not
+specified, the mirror list will not need to be signed if mirror lists are
```

Comment:
line 441: "the mirror list will not need to be signed" add "even" at the end of this phrase